packages/cli/src/commands/devHandler.js

```diff
@@ -178,7 +197,16 @@ export const handler = async ({
   const jobs = {
     api: {
       name: 'api',
-      command: `yarn cross-env NODE_ENV=development NODE_OPTIONS="${getDevNodeOptions()}" yarn nodemon --quiet --watch "${redwoodConfigPath}" --exec "yarn rw-api-server-watch --port ${apiAvailablePort} ${getApiDebugFlag()} | rw-log-formatter"`,
+      command: [
+        `yarn cross-env NODE_ENV=development NODE_OPTIONS="${getDevNodeOptions()}"`,
+        '  yarn nodemon',
+        '    --quiet',
+        `    --watch "${redwoodConfigPath}"`,
+        '    --exec "yarn rw-api-server-watch',
+        `      --port ${apiAvailablePort}`,
+        `      ${getApiDebugFlag()}`,
+        '      | rw-log-formatter"',
```

Comment:
just a stylistic comment: any reason for the indentation?

Reply:
Yeah, to see what option belong to what command. `--port` for example is difficult to know that it "belongs" to `rw-api-server-watch` if there's no indentation